mindmeld/components/entity_recognizer.py

```diff
@@ -41,17 +42,20 @@ class EntityRecognizer(Classifier):
     CLF_TYPE = "entity"
     """The classifier type."""
 
-    def __init__(self, resource_loader, domain, intent):
+    def __init__(self, resource_loader, domain, intent, sys_recognizer=None):
         """Initializes an entity recognizer
 
         Args:
             resource_loader (ResourceLoader): An object which can load resources for the classifier
             domain (str): The domain that this entity recognizer belongs to
             intent (str): The intent that this entity recognizer belongs to
+            sys_recognizer (SystemEntityRecognizer): The system entity recognizer, default to
+                Duckling
         """
         super().__init__(resource_loader)
         self.domain = domain
         self.intent = intent
+        self.sys_recognizer = sys_recognizer or DucklingRecognizer.get_instance()
```

Comment:
I think `sys_recognizer ` should be renamed to `system_entity_recognizer`

Reply:
done